cfme/common/provider.py

```diff
@@ -672,6 +680,19 @@ def load_all_provider_vms(self):
             sel.click(details_page.infoblock.element("Relationships", self.vm_name))
             return True
 
+    def load_all_provider_nodes(self):
```

Comment:
We really want to try getting away from load_functions, it's what navmazing and the navigation is for, could you perhaps instead make a navmazing destionation called ```'AllNodes'```